### ISSUE TITLE
Confusion about number of images

### ISSUE THREAD

**Issue by MrMonk3y** [anjany/verse#6]
First of all, thank you for compiling this data set! 

I have some inconsistencies with the number of samples. 
Verse2020 at https://osf.io/t98fz/ states that there are 300 images. This is the case with the MICCAI version. However, when downloading the subject-based data structure, there are 214. 61 training samples, 80 test samples, and 73 validation samples. It is also unclear to me why the training set is smaller than the testing and validation set. 

I started comparing the MICCAI and subject-based datasets and found that most of the samples are the same. But MICCAI seems to have some samples with numbers below 500, which I also found in verse2019. For example, the verse412 image seems to show up in both 2019 and MICCAI 2020. This seems odd to me, as it is stated that _"... is an extension of the verse19 dataset ..."_.

Do verse2019 and verse2020 have images in common? I was not under the impression that Verse2019 was fully included in Verse2020. And could you perhaps clarify on the number of samples?

**Comment by anjany** [comment]
1. Yes, there is an overlap between VerSe19 and VerSe20. However, not all of VerSe19 are in VerSe20. Please refer to #1 .
2. In subject-based VerSe 2020, the subjects already in VerSe19 are not included. Hence the discrepancy. So, if you copy over the "overlapping subjects" from the subject-based VerSe 2019, you should be good. (Overlap is the VerSe IDs < 500, as you figured out.)

Why do we do this? If you are benchmarking, use the MICCAI-based OSF versions. If you want to use VerSe for other purposes, combine the subject-based versions of both VerSe19 and VerSe20 and work.

**Comment by MrMonk3y** [comment]
Thanks a lot. That clarifies it. I'll close it if that's okay.
